Sources/Testing/Expectations/Expectation.swift

```diff
@@ -62,3 +62,65 @@ public struct ExpectationFailedError: Error {
   /// The expectation that failed.
   public var expectation: Expectation
 }
+
+// MARK: - Snapshot
+
+extension Expectation {
+  /// A serializable type describing an expectation that has been evaluated.
+  @_spi(ExperimentalSnapshotting)
+  public struct Snapshot: Sendable, Codable {
+    /// The source code of the expression evaluated by this expectation, if
+    /// available at compile time.
+    public var sourceCode: String?
```

Comment:
We have discussed _eventually_ making our `SourceCode` type conform to `Codable` itself, but we don't feel that type's design is sufficiently nailed down to do that right now. Since it's possible/likely that we will eventually make `SourceCode` codable itself, whenever we do that we'll probably want to use the name `sourceCode` for that property here.

Given that, should we name this `String`-typed property sourceCode**Description**, or something other than plain `sourceCode`?